perf/benchmark/runner/fortio.py

```diff
@@ -209,7 +208,7 @@ def sync_fortio(url, table, selector=None, promUrl="", csv=None, csv_output="",
 
     if table:
         return write_table(table, datafile)
-
+    os.system("rm -rf {temp_dir_path}".format(temp_dir_path=temp_dir_path))
```

Comment:
should this be under a flag so its not always deleting?

Reply:
talked with Shuayb about some options on what he was trying to achieve with the temp directory removal.